modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -381,6 +408,27 @@ private <T extends RaftGroupService> CompletableFuture<T> startRaftGroupNodeInte
                 : raftServiceFactory.startRaftGroupService(nodeId.groupId(), configuration, raftConfiguration, executor, cmdMarshaller);
     }
 
+    private void startRaftGroupNodeInternalWithoutService(
```

Comment:
Should we call this method inside `startRaftGroupNodeInternal` in order to remove code duplication or they are not the same?

Reply:
Yes, thanks, will fix